shell/components/nav/NamespaceFilter.vue

```diff
@@ -87,7 +87,8 @@ export default {
       const t = this.$store.getters['i18n/t'];
       let out = [];
 
-      if (this.currentProduct.customNamespaceFilter) {
+      // TODO: Add return info
```

Comment:
I think it's a good practice to link a todo in comments to a github issue. This way other developers can gather context as to what needs to be done and we can identify if this has been finished.

Reply:
I totally agree with you, still this reason is not sufficient to need a ticket as there's not enough requirements to write down, even if we just want to refactor it (comment above). This should rather be addressed in a next issue, where the functionality is required and needs to be defined to proceed.

I keep it unresolved to know if you agree or if you have anything in mind.

Reply:
I think we're fine here. It sounds like we have some potential future refactoring work but we would need to gather requirements for it. These todos would get identified during that process.